.travis/wait_for_services.rb

```diff
@@ -40,6 +40,6 @@
     end
   end
 end
-puts 'Waiting 2 minutes for agent to complete it\'s first round...'
-sleep 120
+puts 'Waiting 1 minute for agent to complete it\'s first round...'
+sleep 60
```

Comment:
Is there a way to find if the agent already did its first round?
I don't like too much the idea of just waiting and then hope that it is done.